coralproject/talk
# fix: oldest first next unread not correctly enabling and jumping to new comments

## What does this PR do?

This makes it so that new comments coming in for oldest first sort will also enable `Next unread` button and correctly scroll to the next unread comment if it's a new comment.

## What changes to the GraphQL/Database Schema does this PR introduce?

none

## Does this PR introduce any new environment variables or feature flags? 

no

## If any indexes were added, were they added to `INDEXES.md`?

n/a

## How do I test this PR?

Add a new comment, logged in as a user on one stream. Have another stream open and sorted by oldest first and all comments marked as seen. See that `Next unread` button enables itself. Press the button. See that you go to the new comment. Also test when there are some other comments before the `Next unread` in the stream.

Also test out with comments not fully loaded, and check that new comments coming in for newest first sort still work as expected.
 
## How do we deploy this PR?
